### PR TITLE
Understand Hashing in Cryptography

### PR DESCRIPTION
**Pull request by JudyNduati**
#### Preliminary Checks - Formatting and Structure 
- [x] Does your article follow any of the suggested structure formats? - **see [example formats](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md#example-format-structure)**
- [x] Is your article properly formatted in Markdown? - **see [Github Markdown guide](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)**
- [x] Have you used the correct folder and file structure? - **see [Contribution Guidelines](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md)**
- [x] Is your article over 750 words? - **Tool**: [Wordcounter.net](https://wordcounter.net)
- [x] Have you used our preferred heading structure? - **[H3 (###) and up](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#headers)**
- [x] Does your article provide enough value and detail about your topic? - **Articles should be clear, accurate, and fully explained.**
- [x] Can your article be understood by beginners? - **Assume the audience is smart but has no prior exposure to the common terminology in your article.**
- [x] Have you included a hero image that is 600x400 pixels and have the copyright to use it? - **Tools:** [Pixlr Image Editor](https://pixlr.com/e) for resizing and [Unsplash](https://unsplash.com) for Creative Commons images
- [x] In your PR - add a comment that states **["This fixes #(enter your Idea Content Form #)"](https://youtu.be/TKJ4RdhyB5Y?t=278)** to link and close your Idea Content (issue) when the PR is merged.

**NOTE: (1st time contributors - submit ONLY one article to get familiar with the process and style-once published, you can then submit any number of articles.)**
      
#### Grammar Checks
- [x] Have you spellchecked and grammar checked your article? - **Tools:** [Write&Improve](https://writeandimprove.com/?lang=en-GB) - [ProWritingAid](https://prowritingaid.com/) -  [Grammarly](https://grammarly.com) 
- [x] Have you checked your article for readability? - **Tool:** [Hemmingway](http://www.hemingwayapp.com)
- [x] Have you added sources for quotes and images that aren't yours?

#### Technical Checks
- [ ] Are your code snippets properly formatted for syntax highlighting - **see [Syntax guide](https://gohugo.io/content-management/syntax-highlighting)**

### Contribution Guidelines
For first-time contributors and for more details, **see [Contributing Guidelines](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md)**

### CODE REVIEW

**Comment by JudyNduati** [comment]
"This Fixes #876"

**Comment by hectorkambow** [comment]
👋 Thank you for submitting this PR - we will review it ASAP - all PRs go into our queue will be looked over before being approved to be published.

File: articles/understand-hashing-in-cryptography/index.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
+### Introduction
+
+Hashing is a form of computer security that enables data integrity and authentication. Hashing, in most cases, is mistaken with encryption. Hashing is different from encryption. We will learn their differences later in this article. In this article, we will explore and learn more about hashing.
+Hashing is a mathematical algorithm that converts plaintext to a unique text string or a ciphertext. That sounds confusing, right?
+
+Let us understand with the help of an example.
+Well, your name is Daniel, and you do not want to disclose your name to anyone. With the use of a hashing algorithm. You can represent your name differently with a unique text string. It's only you who can tell the unique text or number.
+"Daniel" = FO8230hRq5K4g7nA
```
Comment (lalith1403) [inline articles/understand-hashing-in-cryptography/index.md:8]: Use formatting to represent the important information. 
```txt
"Daniel" = FO8230hRq5K4g7nA
```

**Review by lalith1403 — CHANGES_REQUESTED** [review]
Well written article @JudyNduati. A minor change and this should good to go.

**Comment by JudyNduati** [comment]
Thank you @lalith1403, I made the change you suggested

**Review by lalith1403 — APPROVED** [review]

**Comment by hectorkambow** [comment]
plagiarism check done

**Comment by hectorkambow** [comment]
great job and article @JudyNduati  
cc @lalith1403  🚀

**Pull request merged**